homeassistant/components/coinbase/config_flow.py

```diff
@@ -0,0 +1,148 @@
+"""Config flow for Coinbase integration."""
+import logging
+
+from coinbase.wallet.client import Client
+from coinbase.wallet.error import AuthenticationError
+import voluptuous as vol
+
+from homeassistant import config_entries, core, exceptions
+from homeassistant.const import CONF_API_KEY, CONF_API_TOKEN
+
+from .const import (  # pylint:disable=unused-import
```

Comment:
Should not be needed anymore on latest dev (might need rebase)

```suggestion
from .const import (
```